packages/experimental/Shadow/src/Shadow.tsx

```diff
@@ -7,13 +7,13 @@ import { getShadowTokenStyleSet } from './shadowStyle';
 export const Shadow = stagedComponent((props: ShadowProps) => {
   return (final: ShadowProps, children: React.ReactNode) => {
     const shadowTokenStyleSet = getShadowTokenStyleSet(props.shadowToken);
-    const mergedProps = mergeProps(final, { style: shadowTokenStyleSet.key });
+    const mergedProps = mergeProps(final, { style: shadowTokenStyleSet.ambient });
 
-    return (
-      <View style={shadowTokenStyleSet.ambient}>
-        <View {...mergedProps}>{children}</View>
-      </View>
-    );
+    const childrenArray = React.Children.toArray(children) as React.ReactElement[];
+    const child = childrenArray[0];
+    const childWithKeyShadow = React.cloneElement(child, mergeProps(child.props, { style: shadowTokenStyleSet.key }));
```

Comment:
>(child.props, { style: shadowTokenStyleSet.key } [](http://example.com/codeflow?start=67&length=48)

This would stomp over any styling that was set on the child, I think?

Reply:
Oh nvm, I forgot that mergeProps automatically handles merging styling smartly, too.

Reply:
Yep I tested to make sure that styling on the child is merged correctly! Can also see it in the snapshots